src/Markdown.MAML/Renderer/Markdownv2Renderer.cs

```diff
@@ -70,6 +78,100 @@ private string MamlModelToString(MamlCommand mamlCommand, Hashtable yamlHeader,
+        {
+            //ensure that all node types in the about topic are handeled.
+            List<MarkdownNodeType> AcceptableNodeTypes = new List<MarkdownNodeType>();
+            AcceptableNodeTypes.Add(MarkdownNodeType.Heading);
+            AcceptableNodeTypes.Add(MarkdownNodeType.Paragraph);
+            AcceptableNodeTypes.Add(MarkdownNodeType.CodeBlock);
+            if (document.Children.Any(c => (!AcceptableNodeTypes.Contains(c.NodeType)) ))
+            {
+                throw new NotSupportedException("About Topics can only have heading, parapgrah or code block nodes in their Markdown Model.");
+            }
+
+            //processes all nodes in order
+            foreach (var currentNode in document.Children)
+            {
+                switch (currentNode.NodeType)
+                {
+                    case MarkdownNodeType.Paragraph:
+                        ParagraphNode paragraphNode = currentNode as ParagraphNode;
+                        AddAboutParagraph(paragraphNode);
+                        break;
+                    case MarkdownNodeType.Heading:
+                        HeadingNode headingNode = currentNode as HeadingNode;
+                        AddAboutHeading(headingNode, document);
+                        break;
+                    case MarkdownNodeType.CodeBlock:
+                        CodeBlockNode codeblockNode = currentNode as CodeBlockNode;
+                        AddAboutCodeBlock(codeblockNode);
+                        break;
+                }
+            }
+            
+            return RenderCleaner.FullNormalization(_stringBuilder.ToString());
+        }
+
+        private void AddAboutCodeBlock(CodeBlockNode codeblockNode)
+        {
+            string[] lines = codeblockNode.Text.Split(new string[] {"\r\n"}, StringSplitOptions.None);
+            
+            foreach (string line in lines)
+            {
+                _stringBuilder.Append("    ");
+                _stringBuilder.AppendLine(line);
+            }
+
+            _stringBuilder.AppendLine();
+        }
+
+        private void AddAboutHeading(HeadingNode headingNode, DocumentNode document)
+        {
+            if (document.Children.ElementAt(0) as HeadingNode == headingNode)
+            {
+                _stringBuilder.AppendLine("TOPIC");
+            }
+            else if (headingNode.HeadingLevel == 1)
+            {
+                _stringBuilder.AppendLine(headingNode.Text.ToUpper());
+            }
+            else if (headingNode.HeadingLevel == 2 && document.Children.ElementAt(1) == headingNode)
+            {
+                _stringBuilder.AppendLine("    " + headingNode.Text.ToLower() + "\r\n");
+            }
+            else if (headingNode.HeadingLevel == 2)
+            {
+                _stringBuilder.AppendLine(headingNode.Text);
+            }
+            else
+            {
+                _stringBuilder.AppendLine("    " + headingNode.Text.ToUpper());
```

Comment:
Where would be a good place for that? There is the Resource Files, but I assume that I should either create a new one, or make it a constant for the class.